openshift/release/tektoncd-pipeline-v0.4.0.yaml

```diff
@@ -0,0 +1,423 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: tekton-pipelines
```

Comment:
are we installing to tekton-pipelines? 

cc @siamaksade

Reply:
We probably shouldn’t. Given that there will be controllers for extensions that are not directly Tekton. ‘openshift-pipelines’ namespace preferably

Reply:
this `release` yaml is generated from the upstream one though (automatically)